roles/zooplus/tasks/apache_ad.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Check if apache active directory studio is installed
+  stat:
+    path: "/home/{{ user }}/SW/ApacheDirectoryStudio/ApacheDirectoryStudio"
```

Comment:
In general /opt is the right base folder for additional tools like POSIX

Reply:
I started to use /opt all across the project, thanks for the hint!